src/cmd/test.go

```diff
@@ -73,7 +74,8 @@ func readJobInput() (*opslevel.RunnerJob, error) {
 	}
 	err := viper.ReadInConfig()
 	if err != nil {
-		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
+		var configFileNotFoundError viper.ConfigFileNotFoundError
+		if !errors.As(err, &configFileNotFoundError) {
```

Comment:
GoLand suggested this [since this is considered a "wrapped error"](https://pkg.go.dev/errors#As)

Reply:
This is the only thing i'm concerned about.  I feel like this changes the logic but maybe it doesn't.  I can see it both ways.  Have you tested this manually?

Reply:
@rocktavious here's an example of a Go playground that compares the 2, showing that they are equivalent: https://go.dev/play/p/aOV4482KlDU

If `err` is wrapped, the new check with `errors.As()` will still work, whereas the old one wouldn't.

This more closely matches intended behaviour, if a `configFileNotFound` happened, then the function should keep running, and if something else happened, the function should return. It does not matter if the error was wrapped or unwrapped.